packages/@aws-cdk/aws-apigateway/package.json

```diff
@@ -283,8 +283,9 @@
       "props-default-doc:@aws-cdk/aws-apigateway.MethodOptions.operationName",
       "props-default-doc:@aws-cdk/aws-apigateway.MethodOptions.requestModels",
       "props-default-doc:@aws-cdk/aws-apigateway.MethodOptions.requestValidator",
-      "docs-public-apis:@aws-cdk/aws-apigateway.ResourceBase.url"
+      "docs-public-apis:@aws-cdk/aws-apigateway.ResourceBase.url",
+      "props-physical-name:@aws-cdk/aws-apigateway.TokenAuthorizerProps"
```

Comment:
Why?

Reply:
Cuz I couldn't figure out what this msg  meant -`error: [awslint:props-physical-name:@aws-cdk/aws-apigateway.TokenAuthorizerProps] Every Resource must have a single physical name construction property, with a name that is an ending substring of <cfnResource>Name`.

My best guess was that there should a property in `TokenAuthorizerProps` called `tokenAuthorizerName`, but that didn't fix this either.

And also, we seem to be [excluding it everywhere](https://github.com/aws/aws-cdk/search?l=JSON&q=%22docs-public-apis%22), so I couldn't quite find one that has this done correctly.

Could you tell me how to fix this, so I can avoid reading reflection code in the awslint for the time being?

Reply:
That's not a very legitimate reason... Please look at the linter rule in awslint and follow it. If we have rules that we exclude everywhere, we need to fix this.

https://github.com/aws/aws-cdk/blob/91c4502cb5d18f4f59714b6a30eef54cdd0b5afc/tools/awslint/lib/rules/resource.ts#L221-L229

P.S. the query you posed was for the "docs-public-api" rule, which we should also not exclude.